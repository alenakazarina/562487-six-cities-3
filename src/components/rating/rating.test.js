import React from 'react';
import renderer from 'react-test-renderer';
import Rating from './rating';

const ratings = [3, 5];

describe(`Rating`, () => {
  it(`should render review Rating with 3 stars`, () => {
    const tree = renderer.create(
        <Rating
          prefix={`reviews`}
          rating={ratings[0]}
        />
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });
  it(`should render place card's Rating with 5 stars`, () => {
    const tree = renderer.create(
        <Rating
          prefix={`place-card`}
          rating={ratings[1]}
        />
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
