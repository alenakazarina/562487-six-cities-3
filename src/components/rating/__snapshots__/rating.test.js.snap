// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Rating should render place card's Rating with 5 stars 1`] = `
<div
  className="place-card__rating rating"
>
  <div
    className="place-card__stars rating__stars"
  >
    <span
      style={
        Object {
          "width": "100%",
        }
      }
    />
    <span
      className="visually-hidden"
    >
      Rating
    </span>
  </div>
</div>
`;

exports[`Rating should render review Rating with 3 stars 1`] = `
<div
  className="reviews__rating rating"
>
  <div
    className="reviews__stars rating__stars"
  >
    <span
      style={
        Object {
          "width": "60%",
        }
      }
    />
    <span
      className="visually-hidden"
    >
      Rating
    </span>
  </div>
</div>
`;
